nishant_work/oop.py

```diff
@@ -0,0 +1,80 @@
+#What is OOP? What are the OOP concept for python? explain with programming.
+# Object-oriented Programming, or OOP for short, is a programming paradigm which provides a means of structuring
+# programs so that properties and behaviors are bundled into individual objects. Python is a great programming
+# language that supports OOP. You will use it to define a class with attributes and methods, which you will then call.
+# Python offers a number of benefits compared to other programming languages like Java, C++ or R. It's a dynamic
+# language, with high-level data types. This means that development happens much faster than with Java or C++. It does
+# not require the programmer to declare types of variables and arguments. This also makes Python easier to understand
+# and learn for beginners, its code being more readable and intuitive.
+
+#Inheritance :-A process of using details from a new class without modifying existing class.Inheritance is a way of
+# creating new class for using details of existing class without modifying it. The newly formed class is a derived class
+# (or child class). Similarly, the existing class is a base class (or parent class).
+
+class Bird:
+
+    def __init__(self):
+        print("Bird is ready")
+
+class Penguin(Bird):
+
+    def __init__(self):
+        super().__init__()
```

Comment:
What is super().__init__() does?

Reply:
when we need to initialize the parent class with in the subclass then this function make the process easier...